src/installerncc.cpp

```diff
@@ -223,6 +223,19 @@ IPluginInstaller::EInstallResult InstallerNCC::invokeNCC(IModInterface *modInter
       seString = std::wstring() + L"-se \"" + seVersion + L"\"";
     }
   }
+  // ticket #142, use a temporary working directory where it had previously used the mod installation directory.
+  QDir m_FinalTargetModDirectory = QString::fromStdWString(QDir::toNativeSeparators(modInterface->absolutePath()).toStdWString().c_str());
+  QDir m_TempModWorkingDirectory;
+  QDir m_modDir;

```

Comment:
It would be best to define these in the class header or else don't use the m_ naming convention.